rubrics/impl/src/main/java/org/sakaiproject/rubrics/impl/RubricsServiceImpl.java

```diff
@@ -1522,6 +1523,181 @@ public Map<String, String> transferCopyEntities(String fromContext, String toCon
+                    Element descriptionEl = doc.createElement("description");
+                    criterionEl.appendChild(descriptionEl);
+                    descriptionEl.appendChild(doc.createCDATASection(description));
+                }
+
+                Element ratingsEl = doc.createElement("ratings");
+                criterionEl.appendChild(ratingsEl);
+
+                criterion.getRatings().forEach(rating -> {
+
+                    Element ratingEl = doc.createElement("rating");
+                    ratingsEl.appendChild(ratingEl);
+                    ratingEl.setAttribute("title", rating.getTitle());
+                    ratingEl.setAttribute("points", Double.toString(rating.getPoints()));
+                    String ratingDescription = rating.getDescription();
+                    if (StringUtils.isNotBlank(ratingDescription)) {
+                        Element ratingDescriptionEl = doc.createElement("description");
+                        ratingEl.appendChild(ratingDescriptionEl);
+                        ratingDescriptionEl.appendChild(doc.createCDATASection(ratingDescription));
+                    }
+                });
+            });
+        });
+
+        stack.pop();
+
+        results.append("completed archiving ").append(getLabel()).append(" for site ").append(siteId).append(LINE_SEPARATOR);
+        return results.toString();
+    }
+
+    @Override
+    public String merge(String toSiteId, Element root, String archivePath, String fromSiteId, Map<String, String> attachmentNames, Map<String, String> userIdTrans, Set<String> userListAllowImport) {
+
+        StringBuilder results = new StringBuilder();
+        results.append("begin merging ").append(getLabel()).append(" for site ").append(toSiteId).append(LINE_SEPARATOR);
+
+        String currentUserId = sessionManager.getCurrentSessionUserId();
+
+        if (!root.getTagName().equals(getLabel())) {
+            log.warn("Tried to merge a non <{}> xml document", getLabel());
+            return "Invalid xml document";
+        }
+
+        NodeList rubricNodes = root.getElementsByTagName("rubric");
+
+        for (int i = 0; i < rubricNodes.getLength(); i++) {
+            Element rubricEl = (Element) rubricNodes.item(i);
+
+            String creatorId = currentUserId;
+
+            String originalCreatorId = rubricEl.getAttribute("creator");
+            if (StringUtils.isNotBlank(originalCreatorId)) {
+                try {
+                    userDirectoryService.getUser(originalCreatorId);
+                    creatorId = originalCreatorId;
+                } catch (UserNotDefinedException unde) {
+                }
+            }
+
+            RubricTransferBean rubricBean = new RubricTransferBean();
+            rubricBean.setOwnerId(toSiteId);
+            // TODO: Do we honour the original creator, or the current user?
+            rubricBean.setCreatorId(creatorId);
+            // TODO: Do we honour the original created date, or use now?
```

Comment:
I think now.  if we think precisely, it is now.

Reply:
Ok, now it will be. We only need the original if we want some kind of audit trail